### PR TITLE
Newsletter-3: translate into Chinese

### PR DESCRIPTION
**Pull request by bittomhan**

### CODE REVIEW

**Review by bitschmidty — COMMENTED** [review]
Links and formatting looked ok to me. Ill give @hulatown some time to take a look as well

**Comment by hulatown** [comment]
> Links and formatting looked ok to me. Ill give @hulatown some time to take a look as well

Sure! Will take a look!

File: _posts/zh/newsletters/2018-07-10-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+title: 'Bitcoin Optech Newsletter #3'
+permalink: /zh/newsletters/2018/07/10/
+name: 2018-07-10-newsletter-zh
+slug: 2018-07-10-newsletter-zh
+type: newsletter
+layout: newsletter
+lang: zh
+version: 1
+---
+本周的 Newsletter 包括有关最低费用和即将发布的比特币核心版本的新闻和行动项目，关于 Schnorr 签名提案的特别专题，以及最近在里斯本举行的 Building on Bitcoin 会议记录。
```
Comment (hulatown) [inline _posts/zh/newsletters/2018-07-10-newsletter.md:11]:
```suggestion
本周的 Newsletter 包括有关最低费用和即将发布的 Bitcoin Core 版本的新闻和行动项目，关于 Schnorr 签名提案的特别专题，以及最近在里斯本举行的 Building on Bitcoin 会议记录。
```

File: _posts/zh/newsletters/2018-07-10-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+[pwuille comment]: https://www.reddit.com/r/Bitcoin/comments/8wmj5b/pieter_wuille_submits_schnorr_signatures_bip/e1wwriq/
+
+## 新闻
+
+- **<!--min-fee-discussion-->[讨论][min fee discussion]最低中继费用**：几年前，当比特币价格仅是其当前美元计价的一小部分时，Bitcoin Core 将最低中继费用设定为 1 satoshi/byte（现在为每 vbyte）。随着价格的上涨和其他网络变化，几位开发者讨论了降低最低中继费用的可能性。Gregory Maxwell 正计划向 Bitcoin Core 提交一个 pull request，该请求可能会将此值大致减半（尽管确切金额尚未确定）。
+
+    这可能会包含在 Bitcoin Core 的下个主要版本中。如果是这样，这意味着一旦变更得到完全部署，您可能能够创建更便宜的合并交易。然而，这也意味着，如果您不升级用于检测未确认交易的任何节点，除非您更改默认设置，否则它们可能不会看到具有低费率的未确认交易。这可能会影响您向用户展示的信息。那些节点仍然会在有效区块中看到所有已确认的交易。
+
+    请注意，要在 Bitcoin Core 中将最低中继费用降低到其默认值以下，您需要更改两个设置。下面显示的是 Bitcoin Core 0.16.1 中具有默认值的两个设置；要降低这些值，请将它们都更改为相同的值，但请注意，将它们降低太多（可能低于默认值的 1/10）会使您面临浪费带宽的攻击，并降低您的节点的 BIP152 紧凑块效率。
```
Comment (hulatown) [inline _posts/zh/newsletters/2018-07-10-newsletter.md:61]:
```suggestion
    请注意，要在 Bitcoin Core 中将最低中继费用降低到其默认值以下，您需要更改两个设置。下面显示的是 Bitcoin Core 0.16.1 中具有默认值的两个设置；要降低这些值，请将它们都更改为相同的值，但请注意，将它们降低太多（可能低于默认值的 1/10）会使您面临浪费带宽的攻击，并降低您的节点的 BIP152 致密区块效率。
```

**Review by hulatown — CHANGES_REQUESTED** [review]
Changes suggested

File: _posts/zh/newsletters/2018-07-10-newsletter.md (hunk 1 above)
Comment (bittomhan) [reply _posts/zh/newsletters/2018-07-10-newsletter.md:11]: 非常感谢您，我之前的没有翻译，这里确实是没有注意到

File: _posts/zh/newsletters/2018-07-10-newsletter.md (hunk 2 above)
Comment (bittomhan) [reply _posts/zh/newsletters/2018-07-10-newsletter.md:61]: 非常感谢！专业名词我得有个参考，我向您私信咨询

**Comment by bittomhan** [comment]
I'm very grateful for Jeffery Hu's review. There were indeed some aspects I overlooked, and I will make sure to pay closer attention and conduct thorough checks in the future!

**Review by bitschmidty — APPROVED** [review]
ACK, thank you @bittomhan and @hulatown !

**Pull request merged**